docs/central-backup.rst

```diff
@@ -7,94 +7,43 @@ Having a data backup strategy is a critical part of running a web service like O
 
 If you are an experienced system administrator, you may want to set up your own backups of the PostgreSQL database that contains all of Central's data. One strategy for doing this is to :ref:`configure a separate database server <central-install-digital-ocean-custom-db>` and back up that database server.
 
-You will additionally need to have a backup of Enketo data to be able to restore existing Web Form links. The strict minimum required to do this are Enketo's Redis store and the keys generated in the Enketo configuration. **In general, we recommend making a full system backup.**
+You will additionally need to have a backup of Enketo data to be able to restore existing Web Form links. The strict minimums required to do this are Enketo's Redis store and the keys generated in the Enketo configuration. **In general, we recommend making a full system backup.**
 
-If you don't already have a full system backup in place and don't want to set up your own database backup, Central provides a managed backup system to Google Drive.
+If you don't already have a full system backup in place and don't want to set up your own database backup, Central provides an API endpoint to download a backup of the database.
 
-.. _central-managed-backups:
+.. _central-direct-backups:
 
-Managed backups / Direct backups via API
-----------------------------------------
+Direct Backups via API
```

Comment:
This section remains, but I removed a couple of other sections on this page. In some cases, I added language from those removed sections to this section.